components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/scheduler/NotificationScheduler.java

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2011, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Updated license header to 2015 in commit https://github.com/wso2/carbon-business-process/commit/c037c2607734679028dc96c59761857176b6dfdb